buildenv/jenkins/openjdk_tests

```diff
@@ -235,9 +238,26 @@ timestamps{
                             }
                         }
                         if (!isNodeIdle) {
-                            println "Cannot find any idle nodes. Starting dynamic vm"
-                            LABEL = LABEL.minus("ci.role.test&&")
-                            LABEL += '&&ci.agent.dynamic'
+                            // DynamicAgents and DockerAgents shouldn't contain the same label
+                            if (params.CLOUD_PROVIDER != null && params.CLOUD_PROVIDER in dynamicAgents) {
+                                LABEL = LABEL.minus("ci.role.test&&")
+                                LABEL += '&&ci.agent.dynamic'
+                                println "Cannot find any idle nodes. Starting dynamic vm, nodeLabel: '${LABEL}', dynamicLabel: '${params.CLOUD_PROVIDER}'"
+                            } else if (params.CLOUD_PROVIDER != null && params.CLOUD_PROVIDER in dockerAgents) {
```

Comment:
Are those two cases `if (params.CLOUD_PROVIDER != null && params.CLOUD_PROVIDER in dockerAgents)` and `else if (dockerAgents.size() >= 1)` same? For both `dockerAgentLabel=default`?